packages/victory-core/src/types/callbacks.ts

```diff
@@ -51,3 +51,5 @@ export type PaddingOrCallback = number | BlockProps | VictoryPaddingCallback;
 export type OrientationOrCallback =
   | OrientationTypes
   | VictoryOrientationCallback;
+
+export type ValueOrCallback<TValue> = TValue | ((args: CallbackArgs) => TValue);
```

Comment:
IMO, I find the `xxxOrCallback` types to be very confusing ... like there's `StringOrCallback` and `StringOrNumberOrCallback` and `NumberOrCallback`.

This new utility can replace all of these, like:
```diff
-    angle?: StringOrNumberOrCallback;
+    angle?: ValueOrCallback<string | number>;
```

If we decide to go this route, internally, then maybe we could mark the old ones `@deprecated`.  But I'll leave that for another PR.